examples/olm/catalog-create-subscribe.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   type: appregistry
   endpoint: https://quay.io/cnr
   registryNamespace: hekumar
-  displayName: "redhat stroage operators"
+  displayName: "Red Hat Storage  operators"
```

Comment:
Should there be two spaces between "Storage" and "operators" ?

Reply:
nope.